theseus/__init__.py

```diff
@@ -16,11 +16,15 @@
     HuberLoss,
     Objective,
     RobustCostFunction,
+    GNCRobustCostFunction,
+    GNCRobustLoss,
+    HingeLoss,
     RobustLoss,
     ScaleCostWeight,
     Variable,
     Vectorize,
     WelschLoss,
+    GemanMcClureLoss,
```

Comment:
Has isort not run here?

Reply:
Good eye lol. We have exceptions for this file due to errors with internal repos (the `# usort: skip` a few lines above). @YipuZhao alphabetical order would still be nice.